torch/distributions/log_normal.py

```diff
@@ -0,0 +1,41 @@
+from torch.distributions.normal import Normal
+from torch.distributions import constraints
+from torch.distributions.bijectors import ExpBijector
+from torch.distributions.transformed_distribution import TransformedDistribution
+
+
+class LogNormal(TransformedDistribution):
+    r"""
+    Creates a log-normal distribution parameterized by
+    `mean` and `std`.
+        X~Normal(loc, scale)
+        Y=exp(X)~LogNormal(loc, scale)
+
+    Example::
+
+        >>> m = LogNormal(torch.Tensor([0.0]), torch.Tensor([1.0]))
+        >>> m.sample()  # log-normal distributed with mean=0 and stddev=1
+         0.1046
+        [torch.FloatTensor of size 1]
+
+    Args:
+        loc (float or Tensor or Variable): mean of log of distribution
+        scale (float or Tensor or Variable): standard deviation of log ofthe distribution
+    """
+    params = {'loc': constraints.real, 'scale': constraints.positive}
+    support = constraints.positive
+    has_rsample = True
+
+    def __init__(self, loc, scale):
+        super(LogNormal, self).__init__(Normal(loc, scale), ExpBijector())

```

Comment:
Should this be `[ExpBijector()]`, since `TransformedDistribution` takes a list of bijectors?